oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandlerTest.java

```diff
@@ -254,6 +254,7 @@ void shouldGenerateSuccessfulAuthResponseAndUpliftAsNecessary(
                 ClientSubjectHelper.calculatePairwiseIdentifier(
                         SUBJECT.getValue(), "test.account.gov.uk", SaltHelper.generateNewSalt());
         session.setInternalCommonSubjectIdentifier(expectedCommonSubject);
+        orchSession.setInternalCommonSubjectId(expectedCommonSubject);
```

Comment:
Not your code, but I'm not a huge fan of this name, given we're using it to set in the sessions. Also, it should have `internalCommonSubjectId` in the name. A big grievance with the current state of of authentication-api is the names of the subjects, so I think it's good to address discrepancies as we go.